scalanativelib/src/ScalaNativeModule.scala

```diff
@@ -50,7 +50,8 @@ trait ScalaNativeModule extends ScalaModule { outer =>
     Seq(
       ivy"org.scala-native:tools_2.12:${scalaNativeVersion()}",
       ivy"org.scala-native:util_2.12:${scalaNativeVersion()}",
-      ivy"org.scala-native:nir_2.12:${scalaNativeVersion()}"
+      ivy"org.scala-native:nir_2.12:${scalaNativeVersion()}",
+      ivy"org.scala-native:test-runner_2.12:${scalaNativeVersion()}"
```

Comment:
It looks like this "test" dependency ends up in a non-test project? My assumption would be to add a test dep only in test projects.

Reply:
These are the dependencies only for linking (they are not Scala Native dependencies but Scala JVM 2.12 ones). Even in Scala JS there are test dependencies in the LinkerClasspath:
https://github.com/lihaoyi/mill/blob/c7873b6f2d977324f215209854a1f415e2882622/scalajslib/src/ScalaJSModule.scala#L40

Reply:
Thanks for clarification.